### PR TITLE
Upgraded yarn v4 and bumped dependencies

### PR DESCRIPTION
**Pull request by victorskl**
* Upgraded to ESLint 9
* Bumped postgres-manager stack dependencies
* Removed obsolete version config from docker compose file
* Removed deprecated eslint config files and yarn plugins

### CODE REVIEW

**Review by raylrui — APPROVED** [review]
LGTM! works fine on my side

File: lib/workload/stateless/stacks/postgres-manager/deploy/lambda-build.Dockerfile [hunk 1]
```diff
@@ -7,6 +7,6 @@ COPY package.json yarn.lock .yarnrc.yml ./
 RUN corepack enable
 RUN yarn --version
 
-RUN yarn install --immutable
```
Comment (williamputraintan) [inline lib/workload/stateless/stacks/postgres-manager/deploy/lambda-build.Dockerfile:10]: no more `--immutable`?

**Review by williamputraintan — APPROVED** [review]
LGTM!

**Review by reisingerf — APPROVED** [review]

**Comment by victorskl** [comment]
Alright. This is looking good. Merging...

**Pull request merged**